src/optimalcontrol/include/iDynTree/Constraint.h

```diff
@@ -1,10 +1,10 @@
 /*
- * Copyright (C) 2014,2017 Fondazione Istituto Italiano di Tecnologia
+ * Copyright (C) 2014,2018 Fondazione Istituto Italiano di Tecnologia
  * Authors: Francesco Romano
  * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
  *
  * Originally developed for Prioritized Optimal Control (2014)
- * Refactored in 2017.
+ * Refactored in 2018.
  * Design inspired by
  * - ACADO toolbox (http://acado.github.io)
  * - ADRL Control Toolbox (https://adrlab.bitbucket.io/ct/ct_doc/doc/html/index.html)
```

Comment:
@traversaro not much time ago unified all the iDynTree copyright headings. Should all this code match that format?

Reply:
Please don't make me modify all files manually again :sob:

Reply:
I thought I commended about this: https://github.com/robotology/idyntree/pull/442#discussion_r190507977 . I typically use tools to automatically do this find and replace job, but we can do it once after you finish all other modifications in the PR.

Reply:
Joking :grin: What I understood before was to have a licence everywhere. I did not understand I had to modify also the content. I would copy that then if needed!